### ISSUE TITLE
Add support for the GDPR cookie consent popup confirmation/widget

### ISSUE THREAD

**Issue by zshamrock** [cryogen-project/cryogen#191]
There are multiple integrations available: https://cookie-script.com/demo.html, https://www.cookiehub.com/. Or custom solution can be build into the platform itself.

**Comment by lacarmen** [comment]
Looks like those two platforms just gives you a script to put on your page? Static sites usually don't make use of cookies so I'm not keen on providing support for this through Cryogen. Seems like it should be simple enough to add it to your templates though.

**Comment by zshamrock** [comment]
But what about Google Analytics tracker, and Google Fonts, and other helper services which utilize the cookies and also collect the data about the user?

**Comment by lacarmen** [comment]
Oh interesting - is the popup required for that as well? I'm not in the EU so not super familiar with GDPR.

I still think it would be simpler to just add the relevant code to your templates. :)

**Comment by holyjak** [comment]
For context: Which cookies need this, see https://wikis.ec.europa.eu/display/WEBGUIDE/04.+Cookies => tracking cookies: yes, auth etc cookies no.

EU Data Protection Board guidelines as of 5/2020: https://edpb.europa.eu/sites/edpb/files/files/file1/edpb_guidelines_202005_consent_en.pdf

**Comment by holyjak** [comment]
I fully agree with Carmen that this does not belong in Cryogen proper. I also hope the popups are going to die soon / get much more limited in scope. I suspect that Google Analytics is indeed in scope of GDPR so we should warn the author about the need to do something about this.
